### PR TITLE
[bitnami/airflow] add podManagementPolicy support for workers

### PR DESCRIPTION
**Pull request by iamsudip**
**Description of the change**

add podManagementPolicy support for workers

**Benefits**

It will allow flexibility to bring up or teardown worker stateful set ordered or parallely

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

**Comment by iamsudip** [comment]
@carrodher Can you please have a look!

File: bitnami/airflow/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: airflow
-version: 6.1.8
+version: 6.1.9
```
Comment (miguelaeh) [inline bitnami/airflow/Chart.yaml:3]: This should be a minor bump

**Comment by sudip-moengage** [comment]
@miguelaeh When does CI/CD picks up these changes? Is it automatic?

**Comment by carrodher** [comment]
Hi, it is automatic, yes; but in this case, we are having some issues with the CI/CD unrelated to this PR, we are working on fixing it. Thanks

**Comment by carrodher** [comment]
After fixing an issue in the above commit, the testing pipeline is failing again:
```
$ helm install . --generate-name
Error: template: airflow/templates/statefulset-worker.yaml:12:16: executing "airflow/templates/statefulset-worker.yaml" at <.Values.worker.podManagementPolicy>: nil pointer evaluating interface {}.podManagementPolicy
```

In order to check if it is an issue in the pipeline itself, did you test those changes? Are you able to install the chart with the default options?

**Comment by sudip-moengage** [comment]
@carrodher My bad, I tested with while `podManagemenetPolicy` was global but later moved it inside `airflow.worker` and added the `if` statement and didn't test again.

**Comment by carrodher** [comment]
Launching it, if everything works as expected a new commit should appear updating the container image tags to the latest revision and the chart should be released

**Comment by carrodher** [comment]
The new version of the Airflow chart was released, you can find it by updating the repo:
```
▶ helm repo update
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "bitnami" chart repository
Update Complete. ⎈ Happy Helming!⎈

▶ helm search repo bitnami/airflow
NAME           	CHART VERSION	APP VERSION	DESCRIPTION
bitnami/airflow	6.2.0        	1.10.10    	Apache Airflow is a platform to programmaticall...
```

**Pull request merged**